### DISCUSSION TITLE
Change email and password for organization

### DISCUSSION THREAD

**Discussion by bionernorway** [community/community#84109] [New to GitHub]
Hi. Is it possible to change the email which is used for log in and "controls" the organization? We now use a work-email were the owner of the mail soon leaves the company.

**Comment by rizkikurniaa** [comment]
Yes, you can change the email associated with your GitHub account, including the email used for logging in and managing organizations. Here's how you can do it:

**1. Change Email on GitHub:**

- Log in to your GitHub account.
- In the top right corner of the page, click on your profile picture, and then click on "Settings."
- In the left sidebar, click on "Emails."
- Under the "Primary email address" section, click on "Change email address."
- Follow the prompts to add a new email address, verify it, and set it as the primary email.

**2. Verify the New Email:**

After adding a new email, GitHub will send a verification email to the new address. Open the email and follow the instructions to verify the new email.

**3. Update Organization Email:**

- If you are the owner or have administrative access to any organizations, you should also update the email associated with the organization.
- Go to the organization's page on GitHub.
- Click on "Settings" in the right sidebar.
- Under the "Profile" tab, you can update the organization's email address.

Remember, if the email change is significant, such as when an owner is leaving the company, it's a good practice to review and update other settings, permissions, and credentials associated with the GitHub account and any connected services or workflows.

If you are part of an organization and are not the owner, you may need to coordinate with the current organization owner to make these changes, especially if they involve updating organization-wide settings.

**Reply by bionernorway** [reply]
Thank you!

**Comment by akashdip2001** [comment]
Yes, it is possible to change the email associated with your GitHub account, including the one used for login and organization management. Here's a general guide:

1. **Change Email on GitHub:**
   - Log in to your GitHub account.
   - Click on your profile picture in the top right.
   - Select "Settings."
   - In the left sidebar, click on "Emails."
   - Under "Primary email address," you can add a new email and set it as primary.

2. **Verify the New Email:**
   - After adding the new email, GitHub will send a verification email to that address.
   - Open the verification email and follow the instructions to confirm the new email.

3. **Update Organization Email:**
   - If the email you want to change is associated with an organization, you may need to update the email in the organization's settings as well.
   - Navigate to your organization's page, click on "Settings," and then go to the "Profile" tab to update the email.

4. **Ensure Administrative Rights:**
   - Make sure the new email has administrative rights for the organization to avoid any access issues.

5. **Transfer Ownership (if needed):**
   - If the current email is the owner of the organization, you might need to transfer ownership to the new email.
   - To transfer ownership, go to your organization's "Settings," then "Member," and use the "Transfer ownership" option.

Remember to communicate any changes to relevant team members and update any automated processes or integrations that rely on the GitHub email. If you encounter any issues during this process, GitHub support can provide further assistance.

## Profile
![Profile Image](https://thumbnails.odycdn.com/optimize/s:200:0/quality:95/plain/https://thumbs.odycdn.com/bf58cf4d7ec52c62f02c52e16f150a5b.webp)
**AKASHDIP MAHAPATRA**
"everything is physics bro"  
E=mc²

|[Subscribe on YouTube](https://www.youtube.com/channel/UCxvmp634YDc41xCWOdvWqoQ?sub_confirmation=1) |
|----  |

**Reply by bionernorway** [reply]
Thank you for the reply
